undertow/src/main/java/org/wildfly/extension/undertow/ListenerResourceDefinition.java

```diff
@@ -165,14 +165,19 @@
         REQUEST_COUNT(new SimpleAttributeDefinitionBuilder("request-count", ModelType.LONG)
                 .setUndefinedMetricValue(new ModelNode(0)).setStorageRuntime().build()),
         BYTES_SENT(new SimpleAttributeDefinitionBuilder("bytes-sent", ModelType.LONG)
+                .setMeasurementUnit(MeasurementUnit.BYTES)
                 .setUndefinedMetricValue(new ModelNode(0)).setStorageRuntime().build()),
         BYTES_RECEIVED(new SimpleAttributeDefinitionBuilder("bytes-received", ModelType.LONG)
+                .setMeasurementUnit(MeasurementUnit.BYTES)
                 .setUndefinedMetricValue(new ModelNode(0)).setStorageRuntime().build()),
         ERROR_COUNT(new SimpleAttributeDefinitionBuilder("error-count", ModelType.LONG)
+                .setMeasurementUnit(MeasurementUnit.BYTES)
```

Comment:
I saw Radim's question about this one on the downstream PR; it's a good question.